### ISSUE TITLE
Fix false positive for $_ variable name

### ISSUE THREAD

**Issue by Konafets** [typo3-ci/TYPO3SniffPool#24]
The CGL says that you must use $_ as variable name inside a foreach if you only need the value of an array

```
foreach ($authors as $key => $value)

foreach ($authors as $_ => $value)
```

This interfere with the rule that variable names with underscore are not allowed.
